### PR TITLE
269 cr def img

### LINKED ISSUE #269: Change news teaser default img CR

### PR DESCRIPTION
**Pull request by gusliedke**
fixes #269

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -114,6 +114,11 @@ module.exports = function (grunt) {
           'dist/images/kids-nav-sprite.png': 'sass/themes/rnd/2017/components/kids-nav/images/kids-nav-sprite.png'
         }
       },
+      cr17: { // Use for subthemes
+        files: {
+          'dist/cr/images/news-default-img.jpg': 'sass/themes/cr/2017/components/news-teaser/images/news-default-img.jpg'
+        }
+      },
```
Comment (Heleen-cr) [inline Gruntfile.js]: I think the imagemin:cr17 task needs to be added to the specific cr17 build task as well.

Comment (gusliedke) [reply Gruntfile.js]: I will update CR task to grab this image too, thanks

**Comment by AndyEPhipps** [comment]
@gusliedke, this is still erroring:

`3:50:10 PM: Fatal error: There is already a template with the ID typography.twig`
@Heleen-cr, any ideas? Is this the twig file you added? Did I'm guessing it built fine locally?

**Comment by Heleen-cr** [comment]
@andy I've added a typography twig file in the sr18 theme. Thought that wouldn't conflict with the other themes..

**Comment by AndyEPhipps** [comment]
Seems happy now!

**Pull request merged**